src/Dhhr.KppParser.Service/Models/NPR(kpp).cs

```diff
@@ -107,6 +111,28 @@ public partial class Melding {
             }
         }
         
+        /// <remarks/>
+        [System.Xml.Serialization.XmlAttributeAttribute()]
+        public System.DateTime uttakTidspunkt {
```

Comment:
Er `uttakTidspunkt` og `uttakTidspunktSpecified` felter som `KppParser` skal populere i forbindelse med genereringen av XML-fila?

Dokumentasjonen jeg finner for `uttakTidspunkt` ([FHI-link](https://www.fhi.no/he/npr/registrering-og-rapportering/tidligere-versjoner-av-informasjonsmodell-og-meldinger/)) sier:

> Tidspunkt for start av uttrekk av data til melding.
Bruk:
Følger ISO 8601 standard: YYYY-MM-DDTHH:MM:SS. 
Eksempler:
2022-04-03T23:10:00 betyr at innholdet i denne melding ble startet hentet ut fra institusjonens datasystemer den 3. april 2022 kl 23:10:00.

Reply:
@dajag

Reply:
`uttakTidspunkt` (og `uttakTidspunktSpecified` er i følge dokumentasjonen ikke påkrevde felter. Vi lar derfor være å populere disse.